Point position
String severity
---
Bool received